README.md

```diff
@@ -17,9 +17,14 @@ There are also advanced topics tutorials:
 
 Most of these tutorials are Jupyter notebooks backed by a python3 kernel.  You'll need:
  - python3 with Jupyter.  [Anaconda](https://www.continuum.io/downloads) is highly recommended.
- - `pypif` and `citrination_client` packages:
- 
- ```pip install pypif citrination_client dfttopif```
+ - Additional packages, which can be installed using `pip3`: or `conda` as follows:
+ ```
+ pip3 install -U -r requirements.txt  
```

Comment:
Hmm, on my machine there is no pip, and I thought pip3 was more clear to say "please use Python 3 in the future;" but I don't know anything about software and can change this 😅

Reply:
> I don't know anything about software

I don't think that's true :-P

I think it makes sense to recommend python3, but we explicitly test all these notebooks for python 2 compatibility so that python 2 users don't have to setup a new environment.